apps/services/form-system/infra/form-system.ts

```diff
@@ -4,11 +4,14 @@ import {
   ServiceBuilder,
 } from '../../../../infra/src/dsl/dsl'
 
-export const serviceSetup = (): ServiceBuilder<'services-form-system'> =>
-  service('services-form-system')
-    .image('services-form-system')
-    .namespace('services-form-system')
+export const serviceSetup = (): ServiceBuilder<'services-form-system-api'> =>
+  service('services-form-system-api')
+    .image('services-form-system-api')
+    .namespace('services-form-system-api')
```

Comment:
consider setting the service name once

```suggestion
const serviceName = 'services-form-system-api'
export const serviceSetup = (): ServiceBuilder<typeof serviceName> =>
  service(serviceName)
    .image(serviceName)
    .namespace(serviceName)
```